CreeDictionary/tests/conftest.py

```diff
@@ -14,7 +14,7 @@
 
 from DatabaseManager.xml_importer import import_xmls
 
-settings.register_profile("default", deadline=timedelta(milliseconds=2000))
+settings.register_profile("default", deadline=timedelta(milliseconds=5000))
```

Comment:
some test took 3000ish ms and triggered DeadlineExceededError in hypothesis

My guess is that Travis CI has downgraded there free tier VMs to garbage cuz GitHub actions exists now. (just trolling)